contracts/src/CrossChainApplications/GETTING_STARTED.md

```diff
@@ -20,245 +20,227 @@ pragma solidity 0.8.18;
+`MyExampleCrossChainMessenger` also needs to implement `ITeleporterReceiver`. First, add the import of this interface:
 
 ```solidity
-// Send a new message to another chain.
-function sendMessage(
-    bytes32 destinationBlockchainID,
-    address destinationAddress,
-    address feeTokenAddress,
-    uint256 feeAmount,
-    uint256 requiredGasLimit,
-    string calldata message
-) external returns (bytes32 messageID) {}
+import {ITeleporterReceiver} from "@teleporter/ITeleporterReceiver.sol";
 ```
 
-`MyExampleCrossChainMessenger` also needs to implement `ITeleporterReceiver` by adding the method `receiveTeleporterMessage` that receives the cross-chain messages from Teleporter.
+Then declare that the contract will implement it:
+
+```diff
+  contract MyExampleCrossChainMessenger is
+-     ReentrancyGuard
++     ReentrancyGuard,
++     ITeleporterReceiver
+  {
+```
+
+And then finally add the method `receiveTeleporterMessage` that receives the cross-chain messages from Teleporter.
 
 ```solidity
-// Receive a new message from another chain.
-function receiveTeleporterMessage(
-    bytes32 sourceBlockchainID,
-    address originSenderAddress,
-    bytes calldata message
-) external {}
+    /**
+     * @dev Receive a new message from another chain.
+     */
+    function receiveTeleporterMessage(
+        bytes32 sourceBlockchainID,
+        address originSenderAddress,
+        bytes calldata message
+    ) external {
+
+    }
 ```
 
 Now it's time to implement the methods, starting with `sendMessage`. First, add the necessary imports.
 
 ```solidity
-import {
-    ITeleporterMessenger,
-    TeleporterMessageInput,
-    TeleporterFeeInfo
-} from "@teleporter/ITeleporterMessenger.sol";
 import {SafeERC20TransferFrom, SafeERC20} from "@teleporter/SafeERC20TransferFrom.sol";
-import {ITeleporterReceiver} from "@teleporter/ITeleporterReceiver.sol";
 import {IERC20} from "@openzeppelin/contracts@4.8.1/token/ERC20/IERC20.sol";
-import {ReentrancyGuard} from "@openzeppelin/contracts@4.8.1/security/ReentrancyGuard.sol";
 ```
 
-Next, add a `using` directive in the contract declaration to specify `SafeERC20` as the `IERC20` implementation to use:
+Next, add a `using` directive to the top of the contract body specifying `SafeERC20` as the `IERC20` implementation to use:
 
 ```solidity
-contract ExampleCrossChainMessenger is
-    ReentrancyGuard,
-    ITeleporterReceiver
-{
     using SafeERC20 for IERC20;
-    ...
-}
 ```
 
-Then in `sendMessage` check whether `feeAmount` is greater than zero. If it is, transfer and approve the amount of IERC20 asset at `feeTokenAddress` to the Teleporter Messenger saved as a state variable.
+Then implement the `sendMessage` function with a check for whether `feeAmount` is greater than zero. If it is, transfer and approve the amount of IERC20 asset at `feeTokenAddress` to the Teleporter Messenger saved as a state variable.
```

Comment:
```suggestion
Then add a check to the `sendMessage` function for whether `feeAmount` is greater than zero. If it is, transfer and approve the amount of IERC20 asset at `feeTokenAddress` to the Teleporter Messenger saved as a state variable.
```

The wording here suggests that the fee check fully implements the function, which it does not.